apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -171,7 +171,7 @@ export const FluentDocsPage = () => {
             {primaryStory.name}
           </HeaderMdx>
           <Primary />
-          <ArgTypes of={primaryStory.component} />
+          <ArgsTable of={primaryStory.component} />
```

Comment:
can we create issue about storybook 8 migration and add this there ? Once the issue is created please add comment to this line referencing to it. ty

> The proposed solution is to switch back to the ArgsTable as it's still available in Storybook v7, and switch to ArgTypes after migration to Storybook v8, [where subcomponents support are in place](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#argstable-doc-block-removed)